test/workos/sso/sso_test.exs

```diff
@@ -49,15 +54,15 @@ defmodule WorkOS.SSOTest do
 
     test "returns the expected query string", %{url: %URI{query: query}} do
       params = URI.query_decoder(query) |> Enum.to_list()
-      assert {"domain", "test"} in params
+      assert {"organization", "test"} in params
     end
   end
 
   describe "#get_authorization_url/2 with a provider" do
     setup do
       {:ok, url} =
         SSO.get_authorization_url(%{
-          domain: "test",
+          organization: "test",
```

Comment:
`organization` and `provider` are mutually-exclusive. Since this test block is testing the `provider` case we can remove the `organization` parameter and just keep `provider`.